ark-cli/README.md

```diff
@@ -1,32 +1,5 @@
 # Ark-CLI
 
-### Installation
-
-To compile you will need openssl libraries and headers:
-
-```shell
-# macOS (Homebrew)
-$ brew install openssl@3
-
-# macOS (MacPorts)
-$ sudo port install openssl
-
-# macOS (pkgsrc)
-$ sudo pkgin install openssl
-
-# Arch Linux
-$ sudo pacman -S pkg-config openssl
-
-# Debian and Ubuntu
-$ sudo apt-get install pkg-config libssl-dev
-
-# Fedora
-$ sudo dnf install pkg-config perl-FindBin openssl-devel
-
-# Alpine Linux
-$ apk add pkgconfig openssl-dev
-```
-
```

Comment:
Actually, it is not required anymore. This was included just because `reqwest` requires that the machine compiling the project have `OpenSSL` installed, but this becomes tricky when it is intended to be cross-compiled.

Related to [Can't cross-compile project with OpenSSL](https://users.rust-lang.org/t/cant-cross-compile-project-with-openssl/70922/3)

Reply:
Awesome, thanks!